common/common.scss

```diff
@@ -21,47 +18,28 @@
     .discourse-kanban-container {
       height: auto;
       padding: 10px 0;
+      max-height: 100%;
     }
   }
 }
 .discourse-kanban-container {
   width: 100%;
   height: 70vh;
+  margin-left:40px;
   display: flex;
   overflow-x: scroll;
   padding: 10px 0 5px 0;
 
-  &:after {
-    content: "";
-    position: absolute;
-    right: 0px;
-    width: 10px;
-    height: 100%;
-    background: linear-a(to right, transparent, var(--secondary, $secondary));
-  }
-  &:before {
-    z-index: 100;
-    content: "";
-    position: absolute;
-    left: 0px;
-    width: 10px;
-    height: 100%;
-    background: linear-gradient(
-      to left,
-      transparent,
-      var(--secondary, $secondary)
-    );
-  }
-
```

Comment:
Any reason for removing these gradient edges? We use these in a few places to make it very clear that a panel is scrollable.

Reply:
They seemed to be interfering with the Kanban control buttons, and I couldn't see any point in having them there.

It might work better to just have it on the right.

Reply:
> Any reason for removing these gradient edges? We use these in a few places to make it very clear that a panel is scrollable.

When the "View x new or updated topics" banner came up, they blocked a very small part of the bottom of the banner, so I removed it

Reply:
> Any reason for removing these gradient edges? We use these in a few places to make it very clear that a panel is scrollable.

I've just put them back. This is the result:

![Screenshot 2022-11-20 19 19 12](https://user-images.githubusercontent.com/63566309/202888832-6fbc0d9f-de93-430a-b052-c39a2e90590b.png)

Basically, they do nothing except stuff up the buttons. And they don't seem to be active on the right (where it is actually scrollable). I'd certainly prefer to be rid of them!

Reply:
> > Any reason for removing these gradient edges? We use these in a few places to make it very clear that a panel is scrollable.
> 
> I've just put them back. This is the result:
> 
> ![Screenshot 2022-11-20 19 19 12](https://user-images.githubusercontent.com/63566309/202888832-6fbc0d9f-de93-430a-b052-c39a2e90590b.png)
> 
> Basically, they do nothing except stuff up the buttons. And they don't seem to be active on the right (where it is actually scrollable). I'd certainly prefer to be rid of them!

Try this?

```css
.discourse-kanban-container:before {
---     left: 0px;
+++     left: 39px;
}
```

Reply:
OK - I discovered that the gradient edge was not working on the right of the Kanban container (where we want it) due to this old bug:

![image](https://user-images.githubusercontent.com/63566309/202927798-df1df95a-41e3-4829-a387-31a29319a166.png)

I also note that the gradient isn't applied to the top of the Kanban Lists, just the bottom. To keep consistent with this, I have removed the gradient from the left of the Kanban container for now.

There is still a minor issue, in that the gradient is applied on the right even if there isn't anything to scroll to. And also, it interferes with the scroll bar on the rightmost Kanban list.

So @davidtaylorhq, I'm not sure that messing about with the gradients is really relevant to this PR as it was a pre-existing problem.

Reply:
Is this currently 'good enough'?